src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -643,6 +643,21 @@
       <summary>Creates an BaseExpressionSyntax node.</summary>
     </FactoryComment>
   </Node>
+  <Node Name="OriginalExpressionSyntax" Base="InstanceExpressionSyntax">
```

Comment:
An alternate implementation would be to just have `original` be an ordinary identifier token, like `value` in a property setter, and let semantic analysis take care of it (i.e. bind it to a method).